README.adoc

```diff
@@ -99,13 +99,13 @@ link:https://github.com/eclipse/microprofile-conference[MicroProfile Conference
 
 A complete list of MicroProfile-related repositories within the Eclipse github organization can be found link:https://github.com/eclipse?utf8=%E2%9C%93&q=microprofile[via this link].
 
-Some of the repositories are hosted under the link:https://github.com/microprofile[Microprofile github organization], including the following repos:
+Some of the repositories are hosted under the link:https://github.com/microprofile[MicroProfile github organization], including the following repos:
 
 link:https://github.com/microprofile/microprofile-blog[MicroProfile.io blogs] -- Easy mechanism for updating the entries on the link:http://microprofile.io/blog[MicroProfile blog site] +
-link:https://github.com/microprofile/microprofile-site[Microprofile.io site] -- source code of the application that powers the link:http://microprofile.io[microprofile.io] site +
+link:https://github.com/microprofile/microprofile-site[MicroProfile.io site] -- source code of the application that powers the link:http://microprofile.io[microprofile.io] site +
 
-=== Eclipse MicroProfile and Microprofile.io
-The MicroProfile community has two portals to information.  The main site is the link:https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+=== MicroProfile and MicroProfile.io
+The MicroProfile community has two portals to information.  The main site is the link:https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project].
 In addition to this portal to our community at Eclipse, we will continue to maintain our external site at link:http://microprofile.io[microprofile.io].
```

Comment:
You changed microprofile.io to MicroProfile.io on lines 105 and 107, but not here.  Since the URL is not case sensitive, I'm okay either way.  But, we should be consistent.

Reply:
fixed